manifests/d/dimkohoncharov/Elixir/1.15.7/dimkohoncharov.Elixir.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: dimkohoncharov.Elixir
+PackageVersion: 1.15.7
+PackageLocale: en-US
+Publisher: Dmytro Honcharov
+PackageName: Elixir language
+License: APACHE License
+ShortDescription: Elixir 1.15.7 language for OTP 2.6

```

Descriptions should be about what the application does in general, not what a specific version is
```suggestion
ShortDescription: Elixir is a dynamic, functional language for building scalable and maintainable applications
```